Core.Auditing/IfxAuditLogger.cs

```diff
@@ -113,8 +112,27 @@ public static string FetchIPAddress()
                     break;
                 }
             }
-
             return ipAddress;
         }
+
+        public void LogRequest(ITelemetryClient telemetryClient, TargetResource[] targetResources, CallerIdentity[] callerIdentities, AuditMandatoryProperties auditMandatoryProperties, AuditOptionalProperties auditOptionalProperties = null)
```

Comment:
>LogRequest [](http://example.com/codeflow?start=20&length=10)

This method is super similar to "LogAuthorizationAuditEvent", do you think you can abstract further?